docs/CommandFormatStandards.md

```diff
@@ -93,11 +93,11 @@ Research and usability testing has revealed that examples are the single most im
 
 - Dependencies between options can exist (if you specify one, you must specify the other, for example). This can introduce usability problems. The dependencies with other options should be noted in the option description.
 
-- The arguments/values for an `option`  can sometimes take wildcards (for example, a data-set name). The wildcard symbol is generally the asterisk * character. When building commands, consider using asterisk as the standard.
+- The arguments/values for an `option` can sometimes take wildcards (for example, a data-set name). The wildcard symbol is generally the asterisk (*) character. When building commands, consider using asterisk as the standard.
```

Comment:
in the Zowe Docs style guide, **data sets** is not hyphenated. Suggesting to do the same here for consistency.